src/Equinor.ProCoSys.IPO.Domain/AggregateModels/InvitationAggregate/IInvitationRepository.cs

```diff
@@ -10,5 +10,7 @@ public interface IInvitationRepository : IRepository<Invitation>
         void RemoveParticipant(Participant participant);
         void RemoveAttachment(Attachment attachment);
         void RemoveComment(Comment comment);
+        void MoveCommPkg(string fromProject, string toProject, string commPkgNo, string description);
```

Comment:
Is/Can a commpkg ever be moved?

Reply:
Yes it can
![image](https://user-images.githubusercontent.com/19545528/107956110-f8865080-6f9e-11eb-8680-21fb11b9204a.png)